<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6" rowstretch="0,0,0,0,0,0,0,0,0,0,4,0,0">
     <item row="9" column="0" rowspan="2" colspan="4">
      <widget class="QGroupBox" name="group_tabla">
       <property name="styleSheet">
        <string notr="true">#group_tabla{
	background-color: #F2F6FD;
	}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout" rowstretch="10,800,0,0" columnstretch="4,3">
        <item row="1" column="0" rowspan="2">
         <widget class="QTableView" name="tbl_ingresos">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="Totales">
          <item>
           <widget class="QLabel" name="label_total_cant">
            <property name="text">
             <string>Total de Articulos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tot_cant">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_total_costo">
            <property name="text">
             <string>Costo Total</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tot_cost">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Articulos a Ingresar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="3">
         <widget class="QTableView" name="tbl_articulos">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Art√≠culos disponibles</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" rowspan="2" colspan="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>INGRESOS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="4">
      <layout class="QHBoxLayout" name="Botones">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_agregar_art">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">#btn_agregar_art{
	background-color: #196F3D;
	color: #ffffff;
	font-family: 'Ariall';
	font-size: 11px;
	border-radius: 5px;
	}

#btn_agregar_art:hover{
	background-color: #04B431;
	}</string>
         </property>
         <property name="text">
          <string>Agregar Articulos</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../ico/insert_to_shopping_cart.ico</normaloff>../ico/insert_to_shopping_cart.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_guardar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">#btn_guardar{
	background-color: #D68910  ;
	color: #ffffff;
	font-family: 'Ariall';
	font-size: 11px;
	border-radius: 5px;
	}

#btn_guardar:hover{
	background-color: #F1C40F ;
	}</string>
         </property>
         <property name="text">
          <string>Guardar Ingreso</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../ico/save.ico</normaloff>../ico/save.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0" rowspan="3" colspan="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Factura</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2" rowspan="2">
      <widget class="QGroupBox" name="group_factura">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#group_factura{
	background-color: #F2F6FD;
	}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="label_tipo">
          <property name="text">
           <string>Tipo</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="fact_tipo">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_numero">
          <property name="text">
           <string>Numero</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="fact_numero">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_fecha">
          <property name="text">
           <string>Fecha</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDateEdit" name="fact_fecha">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_prefijo_fact">
          <property name="text">
           <string>Prefijo</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="fact_prefijo">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QGroupBox" name="group_remito">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#group_remito{
	background-color: #F2F6FD;
	}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="1" column="0">
         <widget class="QLabel" name="label_tipo_2">
          <property name="text">
           <string>Tipo</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="rem_tipo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_prefijo">
          <property name="text">
           <string>Prefijo</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="rem_prefijo">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_numero_2">
          <property name="text">
           <string>Numero</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="rem_numero">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_fecha_2">
          <property name="text">
           <string>Fecha</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDateEdit" name="rem_fecha">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="group_proveedor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#group_proveedor{
	background-color: #F2F6FD;
	}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_nombre_3">
          <property name="text">
           <string>Nombre Completo</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="prov_proveedor">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Remito</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Proveedor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>prov_proveedor</tabstop>
  <tabstop>rem_prefijo</tabstop>
  <tabstop>rem_numero</tabstop>
  <tabstop>fact_tipo</tabstop>
  <tabstop>fact_prefijo</tabstop>
  <tabstop>fact_numero</tabstop>
  <tabstop>tbl_ingresos</tabstop>
  <tabstop>btn_agregar_art</tabstop>
  <tabstop>btn_guardar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
